addon/helpers/compact.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  get,
+  set,
+  observer,
+  isPresent,
+  computed: { filter }
+} = Ember;
+
+export default Helper.extend({
+  compute([array]) {
+    set(this, 'array', array);
```

Comment:
We should add a guard here for when `array` is not actually an `array` using `isArray`. In this scenario we can just return the value wrapped in an array:

```js
if (!isArray(array)) {
  return [array];
}
```